### PR TITLE
protofy_response shortcut added.

### PR DESCRIPTION
**Pull request by go-ujjwal**

### CODE REVIEW

File: src/protofy_shortcuts.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    :param proto_class: (class) proto class from the source.
+    :return: (dict)
+    """
+    proto_obj = proto_class()
+
+    # Deserialize proto
+    proto_obj.ParseFromString(message)
+
+    # get a dict out
+    item = protobuf_to_dict(proto_obj)
```
Comment (neerajkoulgoibibo) [inline src/protofy_shortcuts.py]: maybe we can name the method as   `get_dict_from_proto_message`

and yes last main just

`return  protobuf_to_dict(proto_obj)`

File: src/protofy_shortcuts.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    return item
+
+
+def get_protofy_response(data_dict, proto_class):
```
Comment (neerajkoulgoibibo) [inline src/protofy_shortcuts.py]: Why don't i get proto_message from the dict then, why do i need a proto_class object out. 

and name the method as   `get_proto_message_from_dict` .

aur haan item =  is redundant, just day return proto_obj.SerializeToString()

File: src/protofy_shortcuts.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+__author__ = 'ujjwal tak'
```
Comment (neerajkoulgoibibo) [inline src/protofy_shortcuts.py]: First and foremost update README  to tell about this helper utility.  And can we name the module `util.py`. It is a utlity afterall..

Comment (go-ujjwal) [reply src/protofy_shortcuts.py]: done

Comment (go-ujjwal) [reply src/protofy_shortcuts.py]: I wanted to use just shortcuts as filename.

File: src/protofy_shortcuts.py (hunk 2 above)
Comment (go-ujjwal) [reply src/protofy_shortcuts.py]: done

Comment (neerajkoulgoibibo) [reply src/protofy_shortcuts.py]: yaar methods ke naam to chaneg kar. 

get_proto_message_from_dict


AND 

get_dict_from_proto_message

**Comment by neerajkoulgoibibo** [comment]
:+1:

**Pull request merged**